api/definitions/api_definition.json

```diff
@@ -44,7 +45,8 @@
 											"hours": "1-2 days per week",
 											"required": "One person",
 											"buddying": false,
-											"scope" : "https://drive.google.com/file/d/1j469c-xp8HPYLocTgz5lMYHnnz7TsRVP/view?usp=sharing"
+											"scope" : "https://drive.google.com/file/d/1j469c-xp8HPYLocTgz5lMYHnnz7TsRVP/view?usp=sharing",
+											"sector" : "Health & Social Care"
```

Comment:
Small thing, but this should be the same as line 33 as both are the same project ("PKAVS project website"), just different roles within it